db/schema.rb

```diff
@@ -133,13 +134,13 @@
   end
 
   create_table "recipes", force: :cascade do |t|
-    t.integer "prep_time"
+    t.time "prep_time"
```

Comment:
I think Ariane has changed the data type from **time** to **integer** - if this change goes trough her code gonna break.